app/engine/engine.ts

```diff
@@ -47,19 +47,22 @@ export function getWinningBuzzer(buzzes: Map<string, number>):
       deltaMs: number;
     }
   | undefined {
-  const result = Array.from(buzzes.entries()).reduce(
-    (acc, [userId, deltaMs]) => {
-      if (
-        deltaMs !== CANT_BUZZ_FLAG &&
-        deltaMs < acc.deltaMs &&
-        deltaMs <= CLUE_TIMEOUT_MS
-      ) {
-        return { userId, deltaMs };
-      }
-      return acc;
-    },
-    { userId: "", deltaMs: Number.MAX_SAFE_INTEGER },
-  );
+  const result = Array.from(buzzes.entries())
+    // Sort buzzes by user ID for deterministic results in case of a tie.
+    .sort((a, b) => (a[0] > b[0] ? 1 : -1))
```

Comment:
oh actually, can you sort like this so it's clear that a and b are userIds?

https://github.com/cmnord/jep/blob/2dce42980be3bcefd8d43e7f02914cdcd8bdc1c3/app/components/prompt/buzz.tsx#L74